footer.php

```diff
@@ -2,29 +2,27 @@
 		</div><!-- container footer-widgets -->
 </div><!-- wrapper container -->
 
-		<div class="container" id="foot">
+		<footer class="container" id="foot">
 			<div class="row">
 
-				<div class="span3">
+				<nav class="span3">
```

Comment:
If there is more than one `<nav>` element, each nav needs to be labeled with their purpose. You should either use `aria-label` to add a label directly, or use `aria-labelledby` to create a connection to the labelling element if one exists. See https://www.w3.org/WAI/tutorials/page-structure/labels/#using-aria-labelledby for details.

Reply:
Thanks for the resource! I updated the code to work with this.